.github/workflows/frontend-checks.yml

```diff
@@ -28,8 +28,19 @@ jobs:
       - name: Lint
         run: npm run check
 
+  test:
```

Comment:
moving the test here is related to the issue?

Reply:
correct: this should avoid that the `may-merge` coming from tests workflow conflicts with the `may-merge `coming from checks workflow, since now they are the same workflow